incubator/open-science-ce/open-science-ce/templates/deployment.yaml

```diff
@@ -0,0 +1,218 @@
+        - name: open-science-ce-{{ .Values.Instance }}-osg-configuration
+          mountPath: /tmp/99-local.ini
+          subPath: 99-local.ini
+        - name: open-science-ce-{{ .Values.Instance }}-htcondor-ce-configuration
+          mountPath: /etc/condor-ce/config.d/99-instance.conf
+          subPath: 99-instance.conf
+        {{ if .Values.Certificate.Secret }}
+        - name: open-science-ce-hostcertkeypair-volume
+          mountPath: /etc/grid-security/hostcert.pem
+          subPath: hostcert.pem
+        - name: open-science-ce-hostcertkeypair-volume
+          mountPath: /etc/grid-security/hostkey.pem
+          subPath: hostkey.pem
+        {{ end }}
+        {{ if .Values.HTTPLogger.Enabled }}
+        - name: log-volume
+          mountPath: /var/log/condor-ce
+        {{ end }}
+        {{ if .Values.VomsmapOverride }}
+        - name: open-science-ce-{{ .Values.Instance }}-voms-mapfile
+          mountPath: /etc/grid-security/voms-mapfile
+          subPath: voms-mapfile
+        {{ end }}
+        {{ if .Values.GridmapOverride }}
+        - name: open-science-ce-{{ .Values.Instance }}-grid-mapfile
+          mountPath: /etc/grid-security/grid-mapfile
+          subPath: grid-mapfile
+        {{ end }}
+        ports:
+        - name: htcondor-ce
+          containerPort: 9619
+          protocol: TCP
+        - name: http # needed to get a cert issued from Let's Encrypt
+          containerPort: 80
+          protocol: TCP
+        env:
+        {{ if eq .Values.Networking.ServiceType "HostNetwork" }}
+        - name: _CONDOR_NETWORK_HOSTNAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: CE_HOSTNAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        {{ else if eq .Values.Networking.ServiceType "LoadBalancer" }} 
+        - name: _CONDOR_NETWORK_HOSTNAME
+          value: {{ .Values.Networking.Hostname }}
+        - name: CE_HOSTNAME
+          value: {{ .Values.Networking.Hostname }}
+        {{ end }}
+        - name: CE_CONTACT
+          value: {{ .Values.Site.ContactEmail }}
+        - name: RESOURCE_NAME
+          value: {{ .Values.Site.Resource }}
+        - name: REMOTE_HOST
+          value: {{ .Values.Cluster.RemoteHost }}
+        - name: REMOTE_BATCH
+          value: {{ .Values.Cluster.RemoteBatch }}
+        - name: DEVELOPER
+          {{ if .Values.Developer.Enabled }}  # https://github.com/helm/helm/issues/2848
+          value: "true"
+          {{ else }}
+          value: "false"
+          {{ end }}
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/usr/local/bin/drain-ce.sh"]
+      - name: osg-submit
+        image: opensciencegrid/submit-host:fresh
+        volumeMounts:
+        - name: spool-volume
+          mountPath: /var/lib/condor/spool/
+        - name: condor-token-volume
+          mountPath: /root/tokens/condor_token
```

Comment:
The image expects it at `/root/secrets/token`

Reply:
Could you submit a PR? Feel free to rip apart any of the submit host related things